staff/llorenç-comellas/spotify-template/index.scss

```diff
@@ -0,0 +1,294 @@
+
+$back-color: #141414;
```

Comment:
@llorenscm nice use of variables! You could have a sass file just for these named: varibales.sass  and import to all sass files.